bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -728,6 +739,7 @@ fi
 
 #catch unset variables
 set -u
+unset PERL5LIB
```

Comment:
This is to do what?

Reply:
When running in a cc7 singularity there's a conflict with the version of Perl on CMS Connect. This makes it default to whatever the desired CMSSW version uses.